alcs-frontend/src/app/features/application/applicant-info/application-details/subd-details/subd-details.component.html

```diff
@@ -11,9 +11,7 @@
       <div class="grid-2">
         {{ lot.type }}
       </div>
-      <div class="grid-3">
-        {{ lot.size }}
-      </div>
+      <div class="grid-3">{{ lot.size }} <span *ngIf="lot.size">ha</span></div>
```

Comment:
Lot.size is a number, so this will return false for 0. Is that okay or should this be lot.size !== null?

Reply:
Good point. The backend doesn't actually save the input if user puts a 0 right now, but I will update it as a safeguard.

Reply:
Updated!